release/models/sampling/openconfig-sampling-sflow.yang

```diff
@@ -58,15 +67,8 @@ module openconfig-sampling-sflow {
         for a specific interface.";
     }
 
-    leaf sampling-rate {
```

Comment:
I wonder whether it's better to have something like:

```yang

leaf rate-mode {
    type enumeration {
       enum SINGLE; // single rate for ingress and egress
       enum DIRECTIONAL; // rate for ingress/egress separately.
    }
}
```

and then leave this leaf along with `ingress-rate` and `egress-rate`. The concern I have with the current proposal is that it's rather implicit that you specify `ingress-rate` and this actually also implies egress rate too.

Reply:
The reason for the removal and replacement of ingress/egress separately is it seems most (all?) implementations support ingress only sampling or ingress/egress sampling - I'm not sure the `SINGLE` enum variant above applies but I'd have to see an implementation reference of a single rate applying to both ingress/egress simultaneously.

I was thinking in the case an implementation supports ingress only - then you could deviate out the egress rate as `not-supported`

e.g. https://www.arista.com/en/um-eos/eos-sflow#xx1152507

Welcome to commentary on above: @rolandphung 

In the case an implementation supports ingress + egress, both are available and can be configured independently with no expectation you must configure both

Reply:
Having a separate ingress/egress rate configuration sounds good, we only support ingress configuration only. This looks to be in the global configuration container as well which works well for us.